main/WebServer.cpp

```diff
@@ -3600,25 +3600,26 @@ namespace http {
 						int ID = atoi(sd[0].c_str());
 						std::string Name = sd[1];
 						_eHardwareTypes Type = (_eHardwareTypes)atoi(sd[2].c_str());
-						switch (Type)
+
```

Comment:
For a start a I added the case label indeed. (The switch case label became 102 for Sysfs). It looks like a value larger then 99 makes the switch fail the raspberry compiler (the case never matches). I did see other people mention this problem when I googled it. The "if's" fixed it.